deploy/examples/create-external-cluster-resources.py

```diff
@@ -525,20 +525,23 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
                 entity = "client.csi-cephfs-provisioner-{}".format(cluster_name)
                 cmd_json = {"prefix": "auth get-or-create",
                             "entity": entity,
-                            "caps": ["mon", "allow r", "mgr", "allow rw",
-                                    "osd", "allow rw tag cephfs metadata=*"],
+                            "caps": ["mon", "allow r, profile simple-rados-client-with-blocklist", 
```

Comment:
Here and everywhere below `allow r` is redundant -- `profile simple-rados-client-with-blocklist` already includes it.